model/converter/json/from_domain.go

```diff
@@ -122,6 +128,10 @@ func (fd fromDomain) convertKeyValues(keyValues model.KeyValues) []json.KeyValue
 			value = kv.Bool()
 		case model.Int64Type:
 			value = kv.Int64()
+			if kv.Int64() > jsMaxSafeInteger || kv.Int64() < jsMinSafeInteger {
+				kv.VType = 0
```

Comment:
```suggestion
```

Reply:
I have done the required changes. Could you please review?